sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -100,39 +100,75 @@ private class CosmosRowConverter(
 
     def fromRowToObjectNode(row: Row): ObjectNode = {
```

Comment:
Please add new tests to https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosRowConverterSpec.scala#L636-L670 for the `origin` scenario?

Reply:
And https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosRowConverterSpec.scala#L636-L670, if it makes sense

Reply:
Done (next iteration)